content/SCALE/SCALEUIReference/Shares/SMBSharesScreens.md

```diff
@@ -102,13 +102,13 @@ This table details the options found on the **Purpose** dropdown list.
 {{< truetable >}}
 | Setting | Description |
 |---------|-------------|
-| **No presets** | Select  to retain control over all **Advanced Options** settings.|
-| **Default share parameters** | The default option when you open the **Add SMB** screen and to use for any basic SMB share. |
-| **Basic time machine share** | Select to set up a basic time machine share. |
-| **Multi-User time machine** | Select to set up a multi-user time machine share. |
-|  **Multi-Protocol (NFSv3/SMB) shares**| Select for multi-protocol (NFSv3/SMB) shares. |
-| **Private SMB Datasets and Shares** | Select to use private SMB datasets and shares. |
-| **SMB WORM. Files become read-only via SMB after 5 minutes** | The **SMB WORM** preset only impacts writes over the SMB protocol. Before deploying this option in a production environment, the user should determine whether the feature meets their requirements. |
+| **No presets** | Select  to retain control over all **Advanced Options** settings. Utilizing this option gives users the flexibility to manually configure SMB parameters. |
```

Comment:
Utilize is a formal way of saying Use, and implies using some thing for some other purpose than what it is intended. While not wrong, lets standardize on the word use, especially when describing how a setting works or is used.
```suggestion
| **No presets** | Select  to retain control over all **Advanced Options** settings. This option gives users the flexibility to manually configure SMB parameters. |
```